### PR TITLE
Add siddhi-io-cdc jar and CDCSample

### PR DESCRIPTION
**Pull request by chathuranga95**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: modules/samples/artifacts/CDCSample/CDCSample.siddhi [hunk 1]
```diff
@@ -0,0 +1,69 @@
+/*
```
Comment (Niveathika92) [inline modules/samples/artifacts/CDCSample/CDCSample.siddhi]: Rename the sample as per standard Receive....

File: modules/samples/artifacts/CDCSample/CDCSample.siddhi [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    2) If the Siddhi application starts successfully, the following message is shown on the console
+        * CDCSample.siddhi - Started Successfully!
+Note:
+    If you want to edit this application while it's running, stop the application, make your edits and save the application, and then start it again. 
+
+Testing the Sample:
+    Use MySQL console or Workbench to insert data to the created table.
+        >> insert into SweetProductionTable values('chocolate',100.0);
+    Observe the inserted data is logged on the console.
+    CDCSample : outstm : Event{timestamp=1541486685789, data=[chocolate, 100.0], isExpired=false}
```
Comment (Niveathika92) [inline modules/samples/artifacts/CDCSample/CDCSample.siddhi]: Isn't it LogStream?

File: modules/distribution/pom.xml [hunk 3]
```diff
@@ -616,6 +616,12 @@
                                     <version>${siddhi.io.ibmmq.version}</version>
                                     <type>jar</type>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.wso2.extension.siddhi.io.cdc</groupId>
+                                    <artifactId>siddhi-io-cdc</artifactId>
+                                    <version>${siddhi.io.cdc.version}</version>
```
Comment (Niveathika92) [inline modules/distribution/pom.xml]: Parent pom with version is not committed.

File: modules/samples/artifacts/CDCSample/CDCSample.siddhi (hunk 2 above)
Comment (chathuranga95) [reply modules/samples/artifacts/CDCSample/CDCSample.siddhi]: Fixed with commit: cfaba7960cd2684d7ff80e509622e7384d589ac1

File: modules/distribution/pom.xml (hunk 3 above)
Comment (chathuranga95) [reply modules/distribution/pom.xml]: Fixed with commit: 29bc2bde951f8c2cb19f17f599f918be88b48df5

**Pull request merged**